gluepyter/glue_ydoc.py

```diff
@@ -251,7 +251,7 @@ def add_links_to_contents(self, links: Dict[str, Dict], contents: Dict):
                     list_name = f"list{'' if lists_count < 0 else f'_{lists_count}'}"
                     lists_count += 1
                     link.pop(f"cids{i}_labels", None)
-                    link.pop(f"data{i}", None)
+                    link[f"data{i}"] = link.pop(f"data{i}", None)
```

Comment:
Removing then adding back the key looks surprising, do you mean to do `link.setdefault(f"data{i}")`?

Reply:
:rofl: I thought there was 2 dictionaries...
I updated it, I just had to remove the line.
Thanks @martinRenou